helpers/focus.js

```diff
@@ -173,7 +173,7 @@ export function isFocusable(node, includeHidden, includeTabbablesOnly, includeDi
 
 	if (_isFocusable && !includeHidden) {
 		// only perform visibility check if absolutely necessary
-		if (nodeName !== 'body' && node.offsetParent === null) return false;
+		if (nodeName !== 'body' && getOffsetParent(node) === null) return false;
```

Comment:
Needed to use the helper here as in Safari, dialog button `offsetParent` was always `null`, so focus was never properly moving to them.

Reply:
And of course this helper doesn't return `false` for hidden elements so is entirely useless here. So need to figure out what's up with Safari in this particular case.